tests/utils/datatypes/datatype_tests.py

```diff
@@ -255,8 +271,12 @@ def test_to_json(self, _mock):
             }
         )
 
-        # TODO: remove mock, fix underlying functionality to not
-        # requery for Resource objects yet again in get_display_value().
-        with self.assertNumQueries(1):
-            json = ri_list.to_json(tile, node)
-        self.assertEqual(json, self.mock_display_value)
+        # Was 4, is now 3, should be 1-2 after fixing:
+        # https://github.com/archesproject/arches/issues/11632
+        with self.assertNumQueries(3):
```

Comment:
This was 1 query before because I was mocking all of `compile_json`. But now that I'm only mocking `displayname`, we're hitting the extra queries mentioned in the folllow-up issue. So I left a comment that it was 4 before the commit that caused me to pop the hood on this.